flink-core/src/main/java/org/apache/flink/configuration/StateChangelogOptions.java

```diff
@@ -34,7 +34,8 @@
                     .defaultValue(Duration.ofMinutes(10))
                     .withDescription(
                             "Defines the interval in milliseconds to perform "
-                                    + "periodic materialization for state backend.");
+                                    + "periodic materialization for state backend. "
+                                    + "The periodic materialization will be disabled when the value is negative");
```

Comment:
I think this causes `ConfigOptionsDocsCompletenessITCase` to fail.
Could you regenerate the docs?

Reply:
Sure, you are right. Thanks for remind.